lib/spack/spack/build_environment.py

```diff
@@ -404,6 +404,11 @@ def set_build_environment_variables(pkg, env, dirty):
             if os.path.isdir(pcdir):
                 env.prepend_path('PKG_CONFIG_PATH', pcdir)
 
+    # We force "unsafe build" for packages that require it
+    # when spack is running as root
+    if os.getgid() == 0:
+        env.set("FORCE_UNSAFE_CONFIGURE", "1")
```

Comment:
Thinking about this a bit I'm curious whether this should go in `build_systems/autotools.py`.  Does anything besides autotools use `FORCE_UNSAFE_CONFIGURE`?

Also if we add this here we should consider removing it from our `Dockerfiles` (in `share/spack/docker`) -- that can be a later PR.

@adamjstewart: what do you think?

Reply:
Ummmm perhaps, it should go on autotools, but I dont know if I other software uses it